pkg/controllers/job/helpers/helpers.go

```diff
@@ -34,8 +35,8 @@ const (
 	persistentVolumeClaimFmt = "%s-pvc-%s"
 )
 
-// GetTaskIndex returns task Index.
-func GetTaskIndex(pod *v1.Pod) string {
+// GetTask returns task Index.
+func GetTask(pod *v1.Pod) string {
```

Comment:
And l think this func can be named more clearly, maybe `getPodIndexUnderTask`?

Reply:
Please discuss the name of this function with @william-wang, I think both are ok